tests/utils_tests/test_functional.py

```diff
@@ -130,7 +131,18 @@ def other_value(self):
 
     def test_cached_property_reuse_different_names(self):
         """Disallow this case because the decorated function wouldn't be cached."""
-        with self.assertRaises(RuntimeError) as ctx:
+        type_msg = (
+            "Cannot assign the same cached_property to two different names ('a' and "
+            "'b')."
+        )
```

Comment:
I couldn't get this working for some reason with 3.12.0a7 but I wanted to see whether this message was in this PEP-678 "note" 🤔

Did you see any "notes"?

Reply:
It was changed in 3.12.0b1

Reply:
Ah here's the new notes:

```
(Pdb) ctx.exception.args
("Cannot assign the same cached_property to two different names ('a' and 'b').",)
(Pdb) ctx.exception.__notes__
["Error calling __set_name__ on 'cached_property' instance 'b' in 'ReusedCachedProperty'"]
```